_partials/_devops-cli-service-forks.md

```diff
@@ -0,0 +1,63 @@
+import CLIINSTALL from "versionContent/_partials/_devops-cli-install.mdx";
+
+To manage development forks:
+
+<Procedure> 
+
+<CLIINSTALL />
+
+1. **Fork the $SERVICE_SHORT**
+
+   ```shell
+    tiger service fork  tgrservice --now --no-wait --name bob   
+   ```
+   By default a fork matches the resource of the parent $SERVICE_LONGs. For paid plans use the `cpu_millis` and 
+   `memory_gbs` for dedicated resources. 
```

Comment:
`cpu_millis` and `memory_gbs` are confusing here, since those aren't the names of the CLI flags. I'd change it to:

```suggestion
   By default a fork matches the resource of the parent $SERVICE_LONGs. For paid plans specify `--cpu` and/or `--memory` for dedicated resources.
```